src/main/groovy/org/mockito/release/version/VersionInfo.java

```diff
@@ -20,15 +18,8 @@
     /**
      * Increments version number in the backing object (typically a file)
      * and returns incremented version info instance.
-     *
-     * @param updateNotable if true, the previous version will be included in the notable versions, too.
-     */
-    VersionInfo bumpVersion(boolean updateNotable);
-
-    /**
-     * Returns notable versions
      */
-    Collection<String> getNotableVersions();
+    VersionInfo bumpVersion();
 
     /**
      * Informs if the current version is a notable release
```

Comment:
Great catch. I missed that. It's a part of public API and it needs to be cleaned.